### PR TITLE
Clarify wording about tournament bans for appeal returnees in `Account restrictions`

### PR DESCRIPTION
**Pull request by Iyarita**
Added clarification that most players are tournament banned upon returning after an appeal (the wiki article had said some players)

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by Walavouchey** [comment]
are there any cases where this does *not* happen? @Ephemeralis 

https://osu.ppy.sh/wiki/en/Help_centre/Tournament_bans:

image

image

File: wiki/Help_centre/Account_restrictions/en.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ If we decide to let you return, we will require that you agree to some terms bef
 - **You should not associate with other accounts** including playing on shared computers.
 - **Any major misbehaviour will result in immediate restriction**. This includes major silences.
 
-In some cases, an appeal returnee may be banned from official and badged tournaments, both as a competitor and member of the tournament's team. The duration of this ban is 1 year per offence (if a user is on their second appeal, the ban lasts 2 years). The unrestriction email will clarify the duration of this ban.
+In most cases, an appeal returnee will be banned from official and badged tournaments, both as a competitor and member of the tournament's team. The duration of this ban is 1 year per offence (if a user is on their second appeal, the ban lasts 2 years). The unrestriction email will clarify the duration of this ban.
```
Comment (Walavouchey) [inline wiki/Help_centre/Account_restrictions/en.md:71]: > The duration of this ban is 1 year per offence (if a user is on their second appeal, the ban lasts 2 years).

this information is not present in the [tournament bans article](https://osu.ppy.sh/wiki/en/Help_centre/Tournament_bans) 🤔

Comment (Blooshing) [reply wiki/Help_centre/Account_restrictions/en.md:71]: Tournament bans themself are not appealable, only the restriction/silence -> restriction. So there would be no "second" appeal for strictly a tournament ban, or even a first one. Silences can lead to restrictions if heinous enough, iirc.

NzGJaHM


Don't really see a reason for it to be included. Tournament bans can be added to someone regardless if they've been restricted/silenced.

Comment (Walavouchey) [reply wiki/Help_centre/Account_restrictions/en.md:71]: The line here is saying that for a second appeal (i.e. offence, from a second regular restriction) then the tournament ban from returning from that second restriction would be 2 years, if i'm understanding this correctly.

This information is not present in the tournament bans article.

> Tournament bans can be added to someone regardless if they've been restricted/silenced.

Right but this line is about automatic tournament bans when someone gets their restriction appeal granted.

**Comment by Ephemeralis** [comment]
under standard appeal terms? no, that always comes with a tournament ban. outside of that, there's appeal review discretion and other non standard things that might change it

File: wiki/Help_centre/Account_restrictions/en.md (hunk 1 above)
Comment (Walavouchey) [inline wiki/Help_centre/Account_restrictions/en.md:71]:
```suggestion
Appeal returnees are also banned from official and badged tournaments, both as a competitor and member of the tournament's team. The duration of this ban is 1 year per offence (if a user is on their second appeal, the ban lasts 2 years). The unrestriction email will clarify the duration of this ban.
```

**Review by peppy — DISMISSED** [review]

**Review by Walavouchey — DISMISSED** [review]

**Comment by Iyarita** [comment]
Hi, should I figure out another way to word the involved text? I wanted to update it to align with the information in the tournament bans article, but maybe I should figure out some other way to say it.

**Comment by Walavouchey** [comment]
can be done in a separate pr if necessary - this one somehow got a bit overdue with merging

File: wiki/Help_centre/Account_restrictions/en.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ If we decide to let you return, we will require that you agree to some terms bef
 - **You should not associate with other accounts** including playing on shared computers.
 - **Any major misbehaviour will result in immediate restriction**. This includes major silences.
 
-In some cases, an appeal returnee may be banned from official and badged tournaments, both as a competitor and member of the tournament's team. The duration of this ban is 1 year per offence (if a user is on their second appeal, the ban lasts 2 years). The unrestriction email will clarify the duration of this ban.
+Appeal returnees are also banned from official and badged tournaments, both as a competitor and member of the tournament's team. The duration of this ban is 1 year per offence (if a user is on their second appeal, the ban lasts 2 years). The unrestriction email will clarify the duration of this ban.
```
Comment (Walavouchey) [inline wiki/Help_centre/Account_restrictions/en.md:71]:
```suggestion
Appeal returnees under the standard terms above are also banned from official and badged tournaments, both as a competitor and member of the tournament's team. The duration of this ban is 1 year per offence (if a user is on their second appeal, the ban lasts 2 years). The unrestriction email will clarify the duration of this ban.
```

**Review by peppy — APPROVED** [review]

File: wiki/Help_centre/Account_restrictions/en.md (hunk 2 above)
Comment (Iyarita) [reply wiki/Help_centre/Account_restrictions/en.md]: I like your changes. I think they're more accurate than what I said.

**Pull request merged**